plugins/inputs/aliyuncms/README.md

```diff
@@ -161,6 +172,8 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
 
 Plugin Configuration utilizes [preset metric items references][2]
 
+//TODO Update the documentation here
+
```

Comment:
What update needs to happen here? We don't like to commit TODOs ;)

Reply:
I have adapted the documentation. Thanks for the hint.